test/test_sklearn_compat.py

```diff
@@ -89,9 +89,13 @@ def stable_init(self, sparsity_param=0.01, num_labeled='deprecated',
     dSDML.__init__ = stable_init
     check_estimator(dSDML)
 
-  # This fails because the default num_chunks isn't data-dependent.
-  # def test_rca(self):
-  #   check_estimator(RCA_Supervised)
+  def test_rca(self):
+    def stable_init(self, num_dims=None, pca_comps=None,
+                    chunk_size=2, preprocessor=None):
```

Comment:
I'd use `**kwargs` here

Reply:
I agree that would be better in general, but for here I think it might be useful to simulate dRCA to have the same arguments names as a real RCA, so that if scikit-learn have checks that depend on the arguments of RCA, they will be taken into account, what do you think ? 
Though it's not the case here so maybe yes we can put **kwargs for simplicity

Reply:
I'm not sure how scikit-learn's testing works, but that seems plausible. This is fine as-is.